browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -35,6 +35,8 @@ int OnBeforeURLRequest_StaticRedirectWork(
   static std::vector<URLPattern> allowed_patterns({
     // Brave updates
     URLPattern(URLPattern::SCHEME_HTTPS, "https://go-updater.brave.com/*"),
+    // Brave updates staging
+    URLPattern(URLPattern::SCHEME_HTTPS, "https://laptop-updates-staging.herokuapp.com/*"),
```

Comment:
Not a big deal since this is just for debugging right now, but can we add a way to only allow staging endpoints in non-prod builds?

Reply:
The only problem is that we also rely on this URL for unit tests, but maybe we can add a define for that.

Reply:
Actually, now that I'm looking closer at this code I noticed that it's wrapped in the following:

```
#if !defined(NDEBUG)
#endif
```

So it will only be called in debug code. There's a comment at the end of the block to turn this into a DCHECK eventually, but that would also mean this block is compiled out in release builds.